lib/engine/game/g_18_uruguay/step/destination_bonus.rb

```diff
@@ -8,14 +8,20 @@ class DestinationBonus < Engine::Step::Base
           ACTIONS = %w[destination_connection pass].freeze
 
           def description
-            'Place the Destination Bonus'
+            'Destination Bonus'
           end
 
-          def pass_description
-            'Skip (Destination Bonus)'
+          def log_skip(entity)
+            return if entity.minor?
+            return if entity.corporation == @game.rptla
```

Comment:
See https://github.com/tobymao/18xx/pull/10719#discussion_r1616336339

```suggestion
            return if entity== @game.rptla
```